pkg/vrraw/vrConsts.go

```diff
@@ -0,0 +1,53 @@
+// Package vrraw offers aliases to all VR abbreviations as defined here:
+//
+// http://dicom.nema.org/medical/dicom/current/output/html/part05.html#table_6.2-1
+//
+// Deprecated VRs from older editions are also included
+package vrraw
+
+const (
+	ApplicationEntity   = "AE"
+	AgeString           = "AS"
+	AttributeTag        = "AT"
+	CodeString          = "CS"
+	Date                = "DA"
+	DecimalString       = "DS"
+	DateTime            = "DT"
+	FloatingPointSingle = "FL"
+	FloatingPointDouble = "FD"
+	IntegerString       = "IS"
+	LongString          = "LO"
+	LongText            = "LT"
+	OtherByte           = "OB"
+	OtherDouble         = "OD"
+	OtherFloat          = "OF"
+	OtherLong           = "OL"
+	OtherVeryLong       = "OV"
+	OtherWord           = "OW"
+	PersonName          = "PN"
+	ShortString         = "SH"
+	SignedLong          = "SL"
+	Sequence            = "SQ"
+	SignedShort         = "SS"
+	ShortText           = "ST"
+	SignedVeryLong      = "SV"
+	Time                = "TM"
+	UnlimitedCharacters = "UC"
+
+	// The spec uses both UniqueIdentifier AND UID in the VR Name column for UI, so
+	// we include both here.
```

Comment:
I'm not so sure about this from a stylistic perspective--it might be nice to have one choice per VR, so client code isn't bouncing back and forth between two symbols that are actually the same under the hood (but may not appear that way when reading the code). What do you think?

Reply:
I'll keep the verbose ones, and leave comments about the other names. My original thought was for people new to the DICOM spec, URL / URI might be more familiar, but that's a good point about redundant symbols.